### PR TITLE
Add example how to use mouse

### PR DESCRIPTION
**Pull request by kant2002**
## What does the pull request do?
Provide short guidance how to use mouse

**Scope of this PR:**
- [ ] fix or update to an existing page
- [X] add a new page to the docs

## Checklist

**If this is a new Page**
- [x] Added the new page to [Summary.md](https://docs.gitbook.com/getting-started/git-sync/content-configuration#summary)

**In any case**
- [x] Spell-checking done
- [x] Checked if all hyperlinks work
- [x] Checked if all images are visible

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: docs/input/pointer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Mouse & Pointer devices
+
+Avalonia operates on the abstraction called pointer devices. These devices including, but not limited to mouse, joystick, touchpad etc. Each control in Avalonia provides following events which allow developer subscribe to the pointer device movements, clicks and wheel movements:
```
Comment (maxkatz6) [inline docs/input/pointer.md:3]:
```suggestion
Avalonia operates on the abstraction called pointer devices. These devices including, but not limited to mouse, touch, pen etc. Each control in Avalonia provides following events which allow developer to subscribe to the pointer device movements, clicks and wheel movements: 
```

File: docs/input/pointer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Avalonia operates on the abstraction called pointer devices. These devices including, but not limited to mouse, joystick, touchpad etc. Each control in Avalonia provides following events which allow developer subscribe to the pointer device movements, clicks and wheel movements: 
+- PointerEnter
+- PointerLeave
+- PointerMoved
+- PointerPressed
+- PointerReleased
+- PointerWheelChanged
+
+Example how to react on mouse button pressed.
```
Comment (maxkatz6) [inline docs/input/pointer.md:11]:
```suggestion
Example how to react on pointer button pressed.
```
while "mouse" is correct for 95% of use cases, I would force correct terminology here

File: docs/input/pointer.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+```csharp
+control.PointerPressed += (args) =>
+{
+    var point = args.GetCurrentPoint(control);
```
Comment (maxkatz6) [inline docs/input/pointer.md:16]:
```suggestion
    var point = args.GetCurrentPoint();
```

see below

File: docs/input/pointer.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    if (point.Properties.IsLeftButtonPressed) {
+        // left button pressed
+    }
+    if (point.Properties.IsRightButtonPressed) {
+        // right button pressed
+    }
+}
+```
+
+In the example above, coordinates `x` and `y` are relatively to control origin. If you want coordinates relative to your window, you may pass it to `PointerEventArgs.GetCurrentPoint` method like this: `var pointWindowCoords = args.GetCurrentPoint(window);`
```
Comment (maxkatz6) [inline docs/input/pointer.md:28]:
```suggestion
In the example above, coordinates `x` and `y` are relatively to the window origin. If you want coordinates relative to the specific control, you may pass it to `PointerEventArgs.GetCurrentPoint` method like this: `var pointControlCoords = args.GetCurrentPoint(control);`
```

File: docs/input/pointer.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+{
+    var point = args.GetCurrentPoint(control);
+    var x = point.Position.X;
+    var y = point.Position.Y;
+    if (point.Properties.IsLeftButtonPressed) {
+        // left button pressed
+    }
+    if (point.Properties.IsRightButtonPressed) {
+        // right button pressed
+    }
```
Comment (maxkatz6) [inline docs/input/pointer.md:24]:
```suggestion
    if (point.Properties.IsLeftButtonPressed)
    {
        // left button pressed
    }
    if (point.Properties.IsRightButtonPressed)
    {
        // right button pressed
    }
```

File: docs/input/pointer.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+}
+```
+
+In the example above, coordinates `x` and `y` are relatively to control origin. If you want coordinates relative to your window, you may pass it to `PointerEventArgs.GetCurrentPoint` method like this: `var pointWindowCoords = args.GetCurrentPoint(window);`
+
```
Comment (maxkatz6) [inline docs/input/pointer.md:29]:
```suggestion

# Tapped event args

Each control also has special gesture events: Tapped and DoubleTapped.
Tapped is raised, when pointer was pressed on the control and then released. 
While DoubleTapped is raised after pointer was pressed two times on the same place. Allowed size (distance between first and second "tap") and time (delay between them) depend on the platform and usually is bigger for touch devices. 

```

File: docs/input/pointer.md (hunk 1 above)
Comment (maxkatz6) [reply docs/input/pointer.md:3]: We don't really support joystick now (unless it's emulated as a mouse), and "touchpad" is always emulated as a mouse on all platforms we support. Touch and Pen are unique types of pointer input though.

File: docs/input/pointer.md (hunk 4 above)
Comment (maxkatz6) [reply docs/input/pointer.md:28]: I believe by default users want position relative to the root window, which is returned when you don't provide any control to this method.

File: docs/input/pointer.md (hunk 5 above)
Comment (maxkatz6) [reply docs/input/pointer.md:24]: nit

**Pull request merged**